scripts/download_image.py

```diff
@@ -295,13 +295,10 @@ def accept_game_download_warning(workers: int,
     False
     """
     import textwrap
-    import shutil
-    choice = input_fn('\n'.join(textwrap.wrap(
-            textwrap.dedent(
-                    WARNING_MESSAGE.format(workers, GAME_WORKER_LIMIT)),
-            shutil.get_terminal_size().columns))).lower()
-    while choice not in ['y', 'n']:
-        choice = input_fn('Either enter Y or N as an argument:').lower()
+    choice = input_fn('\n'.join(textwrap.wrap(textwrap.dedent(
```

Comment:
'\n'.join(textwrap.wrap(   need this?
we can keep the WARING_MESSAGE text length short.

workers and limit are only number, length will not be too long
unless user input huge number. btw range check for the options?

Reply:
Can you repull again?